src/PotentialCalculation/SuccessiveOverRelaxation/GPU_kernel_Cartesian3D.jl

```diff
@@ -1,20 +1,22 @@
 @kernel function sor_car_gpu!(
     potential::AbstractArray{T, 4},
-    point_types::AbstractArray{PointType, 4},
-    volume_weights::AbstractArray{T, 4},
-    q_eff_imp::AbstractArray{T, 4},
-    q_eff_fix::AbstractArray{T, 4},
-    ϵ_r::AbstractArray{T, 3},
-    geom_weights::NTuple{3, <:AbstractArray{T, 2}},
-    sor_const::AbstractArray{T, 1},
-    update_even_points::Bool,
-    depletion_handling_enabled::Bool,
-    is_weighting_potential::Bool,
-    only2d::Bool
+    imp_scale::AbstractArray{T, 4},
+    @Const(point_types::AbstractArray{PointType, 4}),
```

Comment:
What does `@Const` do? Not modify the array within the function?

Reply:
https://juliagpu.github.io/KernelAbstractions.jl/dev/api/#KernelAbstractions.@Const

It gives some additional information to the compiler, that those fields are not modified (read-only) during the kernel. 
This information can sometimes improve performance. 
Though, in our case, I did not observe any speed up.